lib/dor/workflow/client.rb

```diff
@@ -90,7 +90,7 @@ def status(druid:, version:)
       # Among other things, a distinct method helps tests mock default logger
       # @return [Logger] default logger object
       def default_logger
-        Logger.new('workflow_service.log', 'weekly')
+        Rails.logger
```

Comment:
This gem doesn't have a dependency on Rails (e.g. robots don't use rails)

Reply:
Ah, shoot... Let me think about this a bit.